ext/zip/php_zip.c

```diff
@@ -1418,9 +1419,13 @@ static ZIPARCHIVE_METHOD(open)
 		RETURN_FALSE;
 	}
 
-	if (!(resolved_path = expand_filepath(filename->val, NULL))) {
-		RETURN_FALSE;
-	}
+    if (!strspn(filename->val,php_spn)) {
```

Comment:
This seems to be only partial fix. What about any other streams? If we want to support streams here, it should support all streams, not only one specific stream.